impeller/entity/contents/filters/gaussian_blur_filter_contents.cc

```diff
@@ -284,8 +287,20 @@ std::optional<Entity> GaussianBlurFilterContents::RenderFilter(
                                 entity.GetClipDepth());  // No blur to render.
   }
 
+  if (!input_snapshot.value().texture->NeedsMipmapGeneration() &&
+      input_snapshot.value().texture->GetMipCount() <= 1) {
```

Comment:
All textures have a mip count of at least 1 (the base mip). Did you mean to do > 1? This is already checked by `NeedsMipmapGeneration()`.

Reply:
To clarify, the mip count is decided when the texture is allocated and and can't ever change.

Reply:
Ahh I see, this is just a noop then?  My reading was that ~NeedsMipmapGeneration means no one has already requested mip maps and MipCount <= 1 means no mips have been calculated.

I was thinking mipmaps were boolean, on or off where we'd calculate them to some predefined rate if they were on.

Reply:
Yeah adding a blit pass won't do anything here yet, since for `ui.Images` they're already generated correctly, and for backdrop filters we aren't allocating higher mip levels if necessary yet. 

Once "Tracking the optimal mip count" is done in https://github.com/flutter/flutter/issues/140949 and we're allocating the `EntityPass` color textures using higher mip counts when necessary, we'll need to perform this blit pass, but with the conditional being:
```
if (input_snapshot.value().texture->NeedsMipmapGeneration()) {
  // ... do a blit pass and generate the mipmap ...
}
```.